README.md

```diff
@@ -27,19 +27,58 @@ If you have a capable (Nvidia, at least 8GB of VRAM) GPU, it is highly recommend
 
 ## Training a model
 
-Open a terminal, change to the directory of the DeepSpeech checkout and run
+The central (Python) script is `DeepSpeech.py` in the project's root directory:
+
+```bash
+$ ./DeepSpeech.py --help
+```
+
+To get the output of this in a slightly better formatted way, you can also look up the option definitions top of `DeepSpeech.py` (line 40++).
```

Comment:
I guess it'd be better not to refer to line numbers as they will likely change and not say in sync with this documentation.